### ISSUE TITLE
Leadfoot 1.6.2 broke support of Appium

### ISSUE THREAD

**Issue by uipoet** [theintern/leadfoot#63]
Using Leadfoot 1.6.1 and Appium 1.4.13 work perfectly with our integration suite of tests.

After upgrading to Leadfoot 1.6.2 or 1.6.3, the createSession promise never resolves nor rejects. The iOS simulator does launch with our app as expected only it then hangs out for a long time waiting for the session promise to resolve before running tests.

```JavaScript
import Server from 'leadfoot/Server';

const server = new Server('http://0.0.0.0:4723/wd/hub');

server
    .createSession({
      app: '../../cordova/platforms/ios/build/emulator/My.app',
      autoWebview: true,
      fixSessionCapabilities: false,
      deviceName: 'iPhone 5s',
      nativeWebTap: true,
      platformName: 'iOS',
      platformVersion: '8.4'
    })
    .then(session => {
        // Can't get here after 1.6.2
    })
    .catch(error => {
        // Can't get here after 1.6.2
    })
    .finally(() => {
        // Can't get here after 1.6.2
    });
```

**Comment by csnover** [comment]
Can you please paste the requests that are being sent to the server?

**Comment by uipoet** [comment]
**1.6.1:**

`POST /wd/hub/session {"desiredCapabilities":{"app":"../../cordova/platforms/ios/build/emulator/My.app","autoWebview":true,"deviceName":"iPhone 5s","nativeWebTap":true,"platformName":"iOS","platformVersion":"8.4"}}`
`POST /wd/hub/session 303`
`GET /wd/hub/session/4e4b54c2-b92d-4bae-9a00-9277f408cb1a`

**1.6.3:**

`POST /wd/hub/session {"desiredCapabilities":{"app":"../../cordova/platforms/ios/build/emulator/My.app","autoWebview":true,"deviceName":"iPhone 5s","nativeWebTap":true,"platformName":"iOS","platformVersion":"8.4"}}`
`POST /wd/hub/session 303`
**Getting the session does not happen.**

**Comment by uipoet** [comment]
@csnover I am happy to provide you the full Appium logs privately, if you believe that will help.

**Comment by csnover** [comment]
Could you please try changing leadfoot/Server.js:52 to `headers: lang.mixin({}, defaultRequestHeaders),` and see if this fixes the problem?

**Comment by uipoet** [comment]
Yes, sir. That fixes it.

**Comment by csnover** [comment]
Fixed by 0ba504f1a0ee0aecd3f13c3176859634ed0b6ea0 in 1.6.4.

**Comment by uipoet** [comment]
Thank you for the speedy fix and release!
